extent.go

```diff
@@ -90,6 +90,19 @@ func (en *ExtentNavigator) Read(offset uint64) (data []byte, err error) {
 		}
 	}()
 
+	// If the inode is not using extents, its data is stored in inode.i_block.
+	if !en.inode.Flag(InodeFlagExtents) {
+		if en.inode.Size() > uint64(len(en.inode.Data().IBlock)) {
+			log.Panicf("Inode size is %v bytes but does not use extents", en.inode.Size())
+		}
+
+		length := en.inode.Size() - offset
+		data = make([]byte, length)
+
+		copy(data, en.inode.Data().IBlock[offset:en.inode.Size()])
+		return
```

Comment:
You need to return `(data []byte, err error)`.

Reply:
Since I am assigning to `data` and err will be nil by default, just return should work right?

Reply:
Yes. You're right.